test/integration/unit/unit_test.go

```diff
@@ -56,8 +60,15 @@ func TestUnitService_Create(t *testing.T) {
 				metadata:    []byte(`{"timezone":"UTC"}`),
 				unitType:    unit.TypeOrg,
 			},
+			setup: func(t *testing.T, params *params, db dbbuilder.DBTX) context.Context {
+				userBuilder := userbuilder.New(t, db)
+				user := userBuilder.Create()
+				params.ownerID = user.ID
+
+				return context.Background()
+			},
 			validate: func(t *testing.T, params params, db dbbuilder.DBTX, result unit.Unit) {
-				require.NotZero(t, result.ID)
+				require.NotZero(t, result.ID) // Need to fix the ID return is zero error
```

Comment:
Consider remove the comment.
```suggestion
				require.NotZero(t, result.ID)
```

Reply:
fixed